kebechet/config.py

```diff
@@ -46,36 +46,80 @@ def from_file(self, config_path: str):
                 content = config_file.read()
 
         try:
-            self._repositories = yaml.safe_load(content).pop('repositories') or []
+            self._repositories = yaml.safe_load(content).pop("repositories") or []
         except Exception as exc:
-            raise ConfigurationError("Failed to parse configuration file: {str(exc)}") from exc
+            raise ConfigurationError(
+                "Failed to parse configuration file: {str(exc)}"
+            ) from exc
+
+    def download_conf_from_url(self, url: str, service: str):
+        params = url.split("/")
+        slug = "/".join(params[3:])
+        user = params[3]
+        repo = params[4]
+        _LOGGER.info(f"{service} service detected")
+        if service == "github":
```

Comment:
Can this be abstracted away and, say, use [this helper function](https://github.com/thoth-station/kebechet/blob/73f9aa272f3be44a7a52b9a914337bcee4154ae8/kebechet/utils.py#L69) which will ensure we will be able to add support for other services easier and in one place?